src/js/main.js

```diff
@@ -39,7 +65,7 @@ Site = {
   setIsMobileWidth: function() {
     var _this = this;
 
-    if (_this.$window.width() >= _this.mobileThreshold) {
+    if (_this.window.width >= _this.mobileThreshold) {
```

Comment:
im confused now. cause line 12 u declare `_this.$window` and then here and on line 48 & 49 u use`_this.window`